src/components/v2/Table/index.tsx

```diff
@@ -7,8 +7,9 @@ import TableContainer from '@mui/material/TableContainer';
 import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
 import Typography from '@mui/material/Typography';
+import { Breakpoint } from '@mui/material';
```

Comment:
This should use the properties of our `BREAKPOINT` constant from the theme instead

Reply:
done